DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -13,7 +13,7 @@
     ViewData[LayoutViewDataKeys.HeaderPathName] ??= dlsSubApplication.HeaderPathName;
   }
   var headerExtension = (string)ViewData[LayoutViewDataKeys.Application];
-  var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";
+  var shouldDisplayHelpMenuItem = dlsSubApplication?.DisplayHelpMenuItem ?? headerExtension != DlsSubApplication.TrackingSystem.HeaderExtension;

```

Comment:
Just to clarify, is the headerExtension check here just for pages that have not been refactored to use the new SetDlsSubApplicationAttribute?

Reply:
Yeah, if the DlsSubApplication is not set into the ViewData by the attribute, we fall back to the old check (except I've changed the string to the Enumeration value so we don't get out of sync). Both these values are used in the layout page for stuff though.